arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -560,9 +560,343 @@
+              "type": "string",
+              "description": "The name of the resource group."
+          },
+          {
+              "name":"queueName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The queue name."
+          },
+          {
+              "name": "namespaceName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The namespace name."
+          },
+          {
+              "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {          
+            "200": {
+                "description": "Queue deleted."
+            }         
+        },
+        "x-ms-long-running-operation": false
+     },
+      "get": {
+          "tags": [
+            "Queues"
+          ],
+          "operationId": "Queues_Get",
+          "description": "Returns the description for the specified namespace.",
+          "externalDocs": {
+              "url": "https://msdn.microsoft.com/en-us/library/azure/mt639380.aspx"
+          },
+        "parameters": [
+          {
+              "name": "resourceGroupName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The name of the resource group."
+          },
+          {
+              "name": "namespaceName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The namespace name."
+          },
+          {
+              "name":"queueName",
+              "in": "path",
+              "required": true,
+              "type": "string",
+              "description": "The queue name."
+          },
+          {
+              "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+            "200": {
+                "description": "Successfully retrieved queue description.",
+                "schema": {
+                    "$ref": "#/definitions/QueueResource"
+                }
+            }
+        }
+      }
     }
   },
   "definitions": {
+     "QueueCreateOrUpdateParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets Queue name."
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets ARM type of the namespace."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets  the location of the resource."
+        },       
+        "properties": {
+          "$ref": "#/definitions/QueueProperties",
+          "description": "Gets or sets properties of the Queues."
```

Comment:
It would be nice if the QueueProperties are flattened. Please apply the x-ms-client-flatten extension.